modules/docs/mdx/5.0-MIGRATION-GUIDE.mdx

```diff
@@ -147,44 +417,84 @@ import {Pagination} from '@workday/canvas-kit-react/pagination';
 
 ### Core
 
-#### Rename Core to Tokens
+#### Remove Labs Core
 
-The distinction between our core and common packages is often unclear and creates confusion around
-what should be imported from where. To help alleviate this and better align with our design
-taxonomy, we've renamed our `core` modules in both Main and Labs to `tokens`. We've also removed the
-`type` default export in the Labs core package in favor of named imports. These changes are listed
-below, all of which are handled by the v5 codemod.
+The Labs `core` package has been removed. The few utilities in that package were either promoted,
+deprecated, or found a better home in another package. These changes are listed below, most of which
+are handled by the v5 codemod.
 
 ##### Automatic Updates
 
-- 🤖 Rename Main `core` import statements to `tokens`
+- 🤖 Move `StaticStates` component to Main `common`
 
-  ```tsx
-  // v4
-  import {colors} from '@workday/canvas-kit-react-core';
+We use `StaticStates` internally for our visual regression tests. It didn't really make sense to
+live in `core`, and it's stable enough to move to Main, so it now lives in `common`.
 
-  // v5
-  import {colors} from '@workday/canvas-kit-react/tokens';
-  ```
+```tsx
+// v4
+import {StaticStates} from '@workday/canvas-kit-labs-react-core';
 
-- 🤖 Rename Labs `core` import statements to `tokens`
+// v5
+import {StaticStates} from '@workday/canvas-kit-react/common';
+```
 
-  ```tsx
-  // v4
-  import {type} from '@workday/canvas-kit-labs-react-core';
+- 🤖 Move `type` tokens to Main `tokens` (formerly `core`)
 
-  // v5
-  import {type} from '@workday/canvas-kit-labs-react/tokens';
-  ```
+This change is described in more detail in the [Type Section](#type), but suffice to say all `type`
+imports will be automatically migrated to the Main `token` package by the codemod.
+
+```tsx
+// v4
+import {type} from '@workday/canvas-kit-labs-react-core';
+
+// v5
+import {type} from '@workday/canvas-kit-react/tokens';
+```
+
+##### Manual Updates
+
+- Deprecate `space` in favor of `Box`
+
+The `space` function was a handy little utility that you could apply to `styled()` components to add
+space style props. However, with the addition of `Box` it is no longer needed. `Box` provides
+`space` style props and much more. While this is a manual migration, the process is fairly
+straight-forward. One important note: The `space` props were a shorthand version of what `Box`
+provides: `pt` maps to `paddingTop`, `mr` maps to `marginRight`, and so on. You can see this in the
```

Comment:
There's probably a way to reword this without having a nested colon.

Reply:
Yep. Agreed. I landed on:

> **Note:** The `space` props use shorthand prop names for what `Box` provides. For example, `pt` maps to `paddingTop`, `mr` maps to `marginRight`, and so on. You can see this in the example below.